modules/diskq/qdisk.c

```diff
@@ -618,10 +634,16 @@ _load_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
 
       msg_info("Disk-buffer state loaded",
                evt_tag_str("filename", self->filename),
-               evt_tag_long("qout_length", self->hdr->qout_pos.count),
-               evt_tag_long("qbacklog_length", self->hdr->qbacklog_pos.count),
-               evt_tag_long("qoverflow_length", self->hdr->qoverflow_pos.count),
-               evt_tag_long("qdisk_length", self->hdr->length));
+               evt_tag_long("number_of_messages", _number_of_messages(self)));
+
+      msg_debug("Disk-buffer internal state",
+                evt_tag_str("filename", self->filename),
+                evt_tag_long("#qout_length", self->hdr->qout_pos.count),
```

Comment:
I would remove the # prefix from all cases. This can confuse users when they found this debug logs.
I understand that this prefix could mean that the value is something which is countable and not a digital unit.

If hard to make a different between which value is a counter or digital unit I would prefer a new naming convention in syslog-ng's debug messages. Of course it would not be a good choice if we know that users are already used this fields as a metric name in their scripts.